dspace-api/src/main/java/org/dspace/embargo/EmbargoServiceImpl.java

```diff
@@ -138,13 +138,11 @@ public DCDate getEmbargoTermsAsDate(Context context, Item item)
                             + result.toString());
         }
 
-        // sanity check: do not allow an embargo lift date in the past.
-        if (liftDate.before(new Date()))
-        {
-            throw new IllegalArgumentException(
-                    "Embargo lift date must be in the future, but this is in the past: "
-                            + result.toString());
-        }
+        /*
+         *  DS-3348 In DSpace 5x and prior, this method returned an IllegalArgumentException if the lift date was in the past.
+         *  This exception was being triggered when performing an AIP restored which is not desirable.
+         *  In XMLUI, the date check (if needed) should be performed in the EmbargoSetter
+         */
```

Comment:
@terrywbrady This looks fine to me. However, I'd recommend either removing this comment, or simplifying it to say something like:

`NOTE: We do not check here for past dates as it can result in errors during AIP restoration. Therefore, UIs should perform any such date validation on input. See DS-3348`

In other words, I don't think we need all the details here about the fact that we used to return a specific exception, etc.